tests/cython/PythonEvalUTest.cxxtest

```diff
@@ -297,4 +299,33 @@ public:
         // Cleanup Python.
         global_python_finalize();
     }
+
+    void testCodeBlockWithNewline()
+    {
+        // Initialize Python.
+        global_python_initialize();
+
+        AtomSpace *as = new AtomSpace();
+        PythonEval::create_singleton_instance(as);
+        PythonEval* python = &PythonEval::instance();
+
+        // Define python functions with newline within the code block
+        string result = python->eval(
+            "def fun(x):\n"
+            "    y = x + 1\n"
+            "\n"
+            "    return y\n"
+            "\n"
```

Comment:
However, if I copy the above into a file, and run it from a file i.e. `python f.py`  then it runs without any errors. So it seems that python is inconsistent with itself, as to what it accepts as valid syntax.

Contents of f.py:
```
def fun(x):
    y = x + 1


    print "foo", y
    return y

fun(1)
print "bar"

```